k9mail/src/main/java/com/fsck/k9/K9.java

```diff
@@ -553,17 +559,17 @@ private void broadcastIntent(String action, Account account, String folder, Mess
                     K9.this.sendBroadcast(intent);
                     if (K9.DEBUG)
                         Log.d(K9.LOG_TAG, "Broadcasted: action=" + action
-                              + " account=" + account.getDescription()
-                              + " folder=" + folder
-                              + " message uid=" + message.getUid()
-                             );
+                                        + " account=" + account.getDescription()
```

Comment:
Please revert unrelated changes. There are a couple more of these whitespace changes.